File: src/main/java/com/linebot/entity/BotUser.java
package com.linebot.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Table(name = "bot_user")
@NoArgsConstructor
@AllArgsConstructor
public class BotUser {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;
    @Column(name = "user_id")
    private String userId;
    @Column(name = "deleted")
    private String deleted;
    @Column(name = "created_date")
    private Date createdDate;
}

File: src/main/kotlin/com/linebot/entity/BotUser.kt
package com.linebot.entity

import java.util.Date
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "bot_user")
data class BotUser(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    var id: Int? = null,
    @Column(name = "user_id")
    var userId: String? = null,
    @Column(name = "deleted")
    var deleted: String? = null,
    @Column(name = "created_date")
    var createdDate: Date? = null
)
